webapp/lib/charts.dart

```diff
@@ -67,6 +67,10 @@ class HistogramChartView {
       scales: new chartjs.LinearScale(yAxes: [
         new chartjs.ChartYAxe()
           ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
+          ..gridLines = (new chartjs.GridLineOptions(color: '#f9f9f9', zeroLineWidth: 0))
+        ],
+        xAxes: [
+          new chartjs.ChartXAxe()..gridLines = (new chartjs.GridLineOptions(color: '#f9f9f9', zeroLineWidth: 0))
```

Comment:
I found `#f9f9f9` to be too light a shade. I think the default for the ticks works well, it was the axis width that was making it very dark and crowded, so I suggest dropping the `color` setting and just keeping `zeroLineWidth: 0`.